agents/plugins/smart

```diff
@@ -108,6 +114,13 @@ if which smartctl > /dev/null 2>&1 ; then
             continue
         fi
 
+        # Exclude removable media
+        if inpath udevadm; then
+            if sh -c "udevadm info /dev/$N | grep -Fxq 'E: ID_CDROM_CD=1'"; then
```

Comment:
Can't we simply use
`if udevadm info /dev/$N | grep -Fxq 'E: ID_CDROM_CD=1'; then`?
I tried an example locally which worked.

Reply:
Should be possible. Haven't tried that, but if it works for you, it'll be in my next commit.